SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -603,13 +603,20 @@ void USpatialNetDriver::OnActorSpawned(AActor* Actor)
 	if (!Actor->GetIsReplicated() ||
 		Actor->GetLocalRole() != ROLE_Authority ||
 		!Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_SpatialType) ||
-		USpatialStatics::IsActorGroupOwnerForActor(Actor))
+		(IsReady() && USpatialStatics::IsActorGroupOwnerForActor(Actor)))
 	{
 		// We only want to delete actors which are replicated and we somehow gain local authority over,
 		// when they should be in a different Layer.
 		return;
 	}
 
+	if (!IsReady())
+	{
+		UE_LOG(LogSpatialOSNetDriver, Warning, TEXT("Spawned replicated actor %s (owner: %s) before the NetDriver was ready. This is not supported. Actors should only be spawned after BeginPlay is called."),
```

Comment:
This might sound like you can't spawn actors during BeginPlay. Also perhaps we should be clearer that this is the BeginPlay that's executed on the GameState, and thus on all other level actors. Any dynamic actor spawn will also have BeginPlay called on it when spawned. Should this say "during or after BeginPlay has been called on the GameState" ? Words are tough.

Reply:
When you say "during or after BeginPlay has been called on the GameState", is this the same as when NotifyBeginPlay is called on the world settings? If so, could instead say after, World->bBegunPlay is set to true?